js/timer.js

```diff
@@ -74,36 +77,34 @@ async function timer(message, channel, time_format) {
     // -----------------------------------
     // Start Loop
     // -----------------------------------
-    const timerID = setInterval(loop, 5 * 60 * 1000);
+
+    const timerID = setInterval(loop, seconds_per_loop * 1000)
```

Comment:
Variable Zeiten sind zwar gut, aber an dieser Stelle muss man leider aufpassen. 
Discord limitiert nämlich das Umbenennen von Channeln für Bots. Ein Bot kann einen Channel nur alle 10 Minuten bis zu zwei mal umbenennen. Das heißt, seconds per loop müsste mindestend 5 sein. Das müsste man noch irgendwo checken oder zuminest in die ReadMe oder Config schreiben